cluster/cluster.go

```diff
@@ -58,6 +58,13 @@ type Cluster struct {
 	UseKubectlDeploy                 bool
 	v3.RancherKubernetesEngineConfig `yaml:",inline"`
 	WorkerHosts                      []*hosts.Host
+	EncryptionConfig                 encryptionConfig
+}
+
+type encryptionConfig struct {
+	RewriteSecrets         bool
+	RotateKey              bool
+	EncryptionProviderFile string
```

Comment:
Instead of just taking the path to the encryption file, can we introduce the object `EncryptionConfiguration` in RKE config? That way the config can be passed directly.

Reply:
This object is not exposed externally and it's used internally only. The `EncryptionConfiguration` type is used in `RancherKubernetesEngineConfig` where it can be added in the cluster yaml file or passed directly.